andela/codango
# #126678329 Redesign Emails

#### What does this PR do?
It makes the emails sent by Codango to be consistent.
#### Description of Task to be completed?
Have emails inherit from one template
#### How should this be manually tested?
The easiest way is by creating a session for Pair Programming and send an invite to an email you have access to then check the email.
#### Any background context you want to provide?
This is how different the email were:

> Popular posts

<img width="1008" alt="screenshot 2016-07-27 11 54 19" src="https://cloud.githubusercontent.com/assets/15629602/17294673/df392cca-5800-11e6-868f-de41bba0213e.png">
> Comments & Likes

<img width="1004" alt="screenshot 2016-07-27 11 54 38" src="https://cloud.githubusercontent.com/assets/15629602/17294675/df49c97c-5800-11e6-9eaf-4e56033396a3.png">
> Session Invite

<img width="1188" alt="screenshot 2016-08-01 14 05 33" src="https://cloud.githubusercontent.com/assets/15629602/17294674/df3bcea8-5800-11e6-8930-c4bad96d19a9.png">
#### What are the relevant pivotal tracker stories?
126678329
#### Screenshots (if appropriate)
<img width="1183" alt="screenshot 2016-08-01 16 08 36" src="https://cloud.githubusercontent.com/assets/15629602/17325085/6476d372-58b2-11e6-8914-f1542bbb87cc.png">
